src/buttons/Button.js

```diff
@@ -123,6 +134,21 @@ const Button = props => {
             backgroundColor && { backgroundColor: backgroundColor },
             borderRadius && { borderRadius },
             !large && styles.small,
+            rounded && {
```

Comment:
@filiphosko instead of passing object here can you move this inside stylesheet.create.